tests/e2e/specs/pacbio_run_edit.js

```diff
@@ -1,22 +1,77 @@
 describe('Pacbio Run Edit view', () => {
+  beforeEach(() => {
+    cy.intercept('/v1/pacbio/runs?include=plate.wells.pools.tube', {
+      fixture: 'tractionPacbioRuns.json',
+    })
+    cy.intercept('/v1/pacbio/runs/7?include=plate.wells.pools.tube', {
+      fixture: 'tractionPacbioRun.json',
+    })
+    cy.intercept(
+      '/v1/pacbio/pools?include=tube,libraries.tag,libraries.request&fields[requests]=sample_name&fields[tubes]=barcode&fields[tags]=group_id&fields[libraries]=request,tag',
+      {
+        fixture: 'tractionPacbioPools.json',
+      },
+    )
+    cy.intercept('/v1/pacbio/runs/7?include=plate.wells.pools.libraries', {
+      fixture: 'tractionPacbioRun.json',
+    })
+    cy.intercept('/v1/pacbio/runs/7', {
+      fixture: 'tractionPacbioRun.json',
+    })
+  })
+
   it('Updates a run successfully', () => {
-    // visit the runs page
-    // click edit existing run
-    // click on well
-    // update movie time
-    // click update (well)
-    // click update (run)
-    // I should see a message run successfully updated
+    cy.intercept('/v1/pacbio/runs/wells/6', {
+      statusCode: 200,
+      body: { data: {} },
+    })
+
+    cy.visit('#/pacbio/runs')
+    cy.get('#editRun-7').click()
+    cy.get('ellipse')
+      .first()
+      .click()
+      .get('#movieTime')
```

Comment:
There's lots of examples of this scattered throughout the tests, but given what I mentioned about get always being global, I'm wondering if its a mistake to chain the methods like this, and if we should always go back to cy when dealing with a new element. It obviously doesn't make a practical difference, but maybe it reads clearer? Possibly one to get @stevieing thoughts on as well, as he seemed to establish the chaining pattern, so perhaps had good reason.